.gitlab/pipeline/test.yml

```diff
@@ -25,6 +25,7 @@ test-linux-stable:
     # "upgrade_version_checks_should_work" is currently failing
     - |
       time cargo nextest run \
+        --filter-expr 'not deps(/polkadot-subsystem-bench/)' \
```

Comment:
I'd move these to a separate pipeline, that allows for CI to provide the same hw specs as for validators and guarantee CPU/mem resources per POD.

Reply:
cc @alvicsam

Reply:
Ok, my bad, we aren't running the tests here 🙈 . let's address this properly in https://github.com/paritytech/polkadot-sdk/issues/3530

Reply:
@AndreiEres could you please add a comment above the command with a brief description why you added the filter or a link to your PR.